VimeoNetworkingExample-iOS/VimeoNetworkingExample-iOSTests/clip_live.json

```diff
@@ -1,14 +1,14 @@
 {
-    "uri": "/videos/224357160",
+    "uri": "/videos/999999",
```

Comment:
Let's use different Id's for different uri's to make sure we don't accidentally pass a test that should fail, e.g. `user.uri == video.uri`

Reply:
How about we also use bogus formats for all uri's to ensure we don't ever risk spamming an actual user / video? e.g. "mock_video_uri_1"

Reply:
This would also make it easier to test, since we can't store these values in constants used everywhere, given they need to be hard coded in JSON, it's easier to type "mock_video_uri_1"